### PR TITLE
fix(Twitter): rename to X.com

### PR DESCRIPTION
**Pull request by rusprice**
## Description 
Rename Twitter to X, rename Tweets to Posts, remove Tweetdeck.

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Draft. Will not work due to translations being renamed, waiting for staff help to update them en masse.

</details>

### CODE REVIEW

**Comment by Bas950** [comment]
@rusprice 
> Need some help from staff to:
>
> * Upload new X images to the PreMiD CDN

Just replace the image urls in the metadata.json file to an imgur link for example, they will auto-update on a cdn by the bot.

> * Make a commit to change the URLs in metadata.json to the new image URLs

Read above

> * Change all the "twitter" translations to "x" on Crowdin

Please make a PR here: https://github.com/PreMiD/Localization/tree/main/src/Presence
1. Rename the twitter.json to x.json
2. Rename the keys of the strings to x instead of twitter.

> * Find/replace "twitter" in all the translations to "X". We will not need to attempt to translate the name because it is a single letter.

Not sure if its in any strings, if it has its own string it can be removed, but if it is in a middle of a string it needs to stay. But will look at your Localization PR when that is ready.

> * Retranslate translations containing "tweet" to "post".

Just like above change this in the Localization PR

**Comment by Bas950** [comment]
@rusprice any updates?

**Comment by rusprice** [comment]
> @rusprice any updates?

Hey, sorry. Computer broke. Just got it repaired today, and will work on the PR more.

**Comment by DarkVillager** [comment]
Goes with https://github.com/PreMiD/Localization/pull/7353

File: websites/X/X/metadata.json
Comment (Bas950) [inline websites/X/X/metadata.json:7]: service and folder names must match

File: websites/X/X/presence.ts
Comment (Bas950) [inline websites/X/X/presence.ts:160]: use an image url

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Review by Bas950 — APPROVED** [review]
LGTM

@theusaf just approve but DONT merge, I will rename the Discord App & Merge the localization pr first once your approval is there.

**Review by Bas950 — CHANGES_REQUESTED** [review]
Actually strings need to be updated, it wont work right now, either the strings need to be renamed to X.com.something in the correct capitalization or here in the metadata we need to change multi lang from true to I think a string or array with the correct naming, need to read docs for that part kek

File: websites/X/X.com/metadata.json
Comment (Bas950) [inline websites/X/X.com/metadata.json:26]:
```suggestion
			"multiLanguage": "x"
```

Comment (Bas950) [reply websites/X/X.com/metadata.json:26]: Think this is the best way of solving it, doesn't make sense to rename all strings again. Especially since the service name includes a dot, will make it confusing. @theusaf @DarkVillager @rusprice agreed?

Comment (theusaf) [reply websites/X/X.com/metadata.json:26]: That makes sense to me.

**Review by Bas950 — APPROVED** [review]
Same like before, dont merge yet

**Review by theusaf — APPROVED** [review]

**Comment by Bas950** [comment]
- [x] Renamed the Discord App to X.com
- [x] Merged localization PR
- [ ] Merged this pr

**Pull request merged**
